README.md

```diff
@@ -76,3 +83,75 @@ attention to:
   Operator](https://github.com/luxas/kubeadm-workshop#deploying-the-prometheus-operator-for-monitoring-services-in-the-cluster)
 - [Setting up the custom metrics adapter and sample
   app](https://github.com/luxas/kubeadm-workshop#deploying-a-custom-metrics-api-server-and-a-sample-app)
+
+FAQs
+----
+
+### Why do my metrics keep jumping between a normal value and a very large number?
+
+You're probably switching between whole numbers (e.g. `10`) and milli-quantities (e.g. `10500m`).
+Worry not!  This is just how Kubernetes represents fractional values.  See the
+[Quantity Values](/docs/walkthrough.md#quantity-values) section of the walkthrough for a bit more
+information.
+
+### Why isn't my metric showing up?
+
+First, check your configuration.  Does it select your metric?  You can find the
+[default configuration](/deploy/custom-metrics-config-map.yaml) in the deploy directory,
+and more information about configuring the adapter in the [docs](/docs/config.md).
+
+Next, check if the discovery information looks right.  You should see the metrics showing
+up as associated with the resources you expect at. `/apis/custom.metrics.k8s.io/v1beta1/`.
```

Comment:
This could maybe be `kubectl get --raw /apis/custom.metrics.k8s.io/v1beta1 | jq .`

Reply:
Agree, at least it should mention `kubectl get --raw` somewhere here.